app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,7 @@
   <head>
     <title><%= yield :title %> - GOV.UK</title>
     <%= stylesheet_link_tag "application" %>
+    <%= javascript_include_tag 'application' %>
```

Comment:
I think that doesn't work in our layouts, this is the way it's done in the frontends I've seen.